eahub/profiles/models.py

```diff
@@ -42,31 +42,31 @@ class CauseArea(enum.Enum):
 
 class ExpertiseArea(enum.Enum):
 
-    MANAGEMENT = 1
-    OPERATIONS = 2
-    RESEARCH = 3
-    GOVERNMENT_AND_POLICY = 4
-    ENTREPRENEURSHIP = 5
-    SOFTWARE_ENGINEERING = 6
-    AI_TECHNICAL_EXPERTISE = 7
-    MATH_QUANT_STATS_EXPERTISE = 8
-    ECONOMICS_QUANTITATIVE_SOCIAL_SCIENCE = 9
-    MOVEMENT_BUILDING = 10
-    COMMUNICATIONS = 11
-    PHILOSOPHY = 12
-    HUMANITIES = 13
-    PSYCHOLOGY = 14
-    PHYSICS = 15
-    MEDICINE = 16
-    OTHER_SCIENCE = 17
-    EVENT_PLANNING = 18
-    FINANCE = 19
-    GRAPHIC_DESIGN = 20
-    JOURNALISM = 21
-    LAW = 22
-    PHILANTHROPY_EARNING_TO_GIVE = 23
-    PUBLIC_SPEAKING = 24
-    RECRUITMENT = 25
+    COMMUNICATIONS = 1
```

Comment:
You can't change which name goes with which number. Those numbers are stored in the database.

Reply:
Then, I understand, that I should add new records to the database and figure out how to sort this list by different means